applications/plugins/MultiThreading/src/TaskSchedulerBoost.cpp

```diff
@@ -189,7 +189,6 @@ namespace sofa
 			mTaskCount		= 0;
 			mFinished		= false;
 			mCurrentStatus = NULL;
-			mTaskMutex.v_ = 0L;
```

Comment:
Ok ... so while digging to fix the failing scenes it seems that this changes generated two new failures. 
This is because now setting v_ to zero leave the spinlock uninitialized and thus in a "locked" state. 
So no-one can grab it. 

I'm not sure how to fix that now as I have an old version of boost.